### DISCUSSION TITLE
Contributor status?

### DISCUSSION THREAD

**Discussion by leeberryman** [wmgeolab/geoBoundaries#1147] [Q&A] [answered]
@DanRunfola Hey is it possible to make me a contributor on the repo?  I can't create branches, so I have to fork it and then all of my issues I'm working on get complicated and added to my pull requests that can only be done one at a time.

**Comment by leeberryman** [comment]
@slfuhrig

**Comment by DanRunfola** [comment, marked as answer]
Yes, done!  We have a contributors team to enable you to branch.

**Reply by leeberryman** [reply]
@DanRunfola I'm sure there is alot going on with the semester winding down and the version 4 release coming up.  Can someone review my pull requests and give me feedback, some have been pending for more than a week, and would like to know if I should anticipate and needed edits before they will be accepted.  Thanks!  Really appreciate what GeoLabs is doing here.

**Comment by DanRunfola** [comment]
Hey Lee -

Sean just let me know he has this on the schedule very shortly, so coming
soon :)

Dan

**Comment by leeberryman** [comment]
@DanRunfola did something change with my permission's on the repo?  "remote: Permission to wmgeolab/geoBoundaries.git denied to leeberryman"

**Reply by DanRunfola** [reply]
No changes - what are you trying to do?  I.e., if you're trying to push straight to the repo it will always fail (even for me) as you have to do a PR --> review.

**Comment by leeberryman** [comment]
Just trying push my local branch to create a remote branch so that I can create the PR.

Lee Berryman
________________________________

**Reply by DanRunfola** [reply]
That should work - let me see if something changed git - side.  Will report back here in a few minutes...

**Reply by DanRunfola** [reply]
Try now - the contributors had originally been set to "Triage", I just upgraded to "Write".  Not sure if that was a github permissions change or something else. The main branch is still protected from everyone, so you shouldn't be able to push to that, but you should definitely be able to create branches.

**Comment by leeberryman** [comment]
Yep that works now.  Thanks.
________________________________
